esmvalcore/cmor/tables/custom/CMOR_n2os.dat

```diff
@@ -0,0 +1,22 @@
+SOURCE: CMIP6
+!============
+variable_entry:    n2os
+!============
+modeling_realm:    atmos
+!----------------------------------
+! Variable attributes:
+!----------------------------------
+standard_name:     mole_fraction_of_nitrous_oxide_in_air
+units:             1e-09
```

Comment:
```suggestion
units:             mol mol-1
```

to align it with the [CMIP6 definition](https://github.com/ESMValGroup/ESMValCore/blob/e2f4d60baee5af51ccf5137a800a0866ee97e215/esmvalcore/cmor/tables/cmip6/Tables/CMIP6_Amon.json#L597).